added files so the web page would build.  Tweaked activity

mwand opened the pull request.

abhutta0 commented:
No need to add all of these.
I have already added the required files in that PR. Once we merge tutorials, then we can merge the PR for Module 01. I added that as  a comment in PR.
We also don't need to publish project overview yet. We should leave that for 2nd week of semester.

mwand commented:
Agreed.  But when I switch to module-01-requirements and say 'make' (which
should build the web page), the build fails.

I added the files only to make the web page build.

The only actual change is to substitute 'State of Massachusetts' for
'Southwest Airlines' and change 'request' to 'obtain'.




On Thu, Aug 31, 2023 at 3:20 PM Adeel A. Bhutta ***@***.***>
wrote:

> No need to add all of these.
> I have already added the required files in that PR. Once we merge
> tutorials, then we can merge the PR for Module 01. I added that as a
> comment in PR.
> We also don't need to publish project overview yet. We should leave that
> for 2nd week of semester.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/neu-se/CS4530-Fall-2023/pull/11#issuecomment-1701645559>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAR3CMDJM4LSWY2U37DSB6LXYDPYTANCNFSM6AAAAAA4GQ7KOQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

abhutta0 commented:
got it. 
Yes built will fail until the tutorials are merged. I will make sure that they are published first before module01. 
I will also reflect the updates to activity.
